configure.ac

```diff
@@ -54,6 +54,9 @@ AC_CHECK_SIZEOF([long long])
 
 gl_LD_VERSION_SCRIPT
 
+AM_ICONV
+AM_GNU_GETTEXT
```

Comment:
There is a tiny warning by autoreconf:
```
root@:~/ding-libs # autoreconf
autoreconf-2.69: configure.ac: AM_GNU_GETTEXT is used, but not AM_GNU_GETTEXT_VERSION
```

Reply:
Correct. If we add that other macro, we gain `intl` and `po` folders and a copy of all the gettext code (for reasons that are beyond me, but apparently make sense to the gettext maintainers). `AM_GNU_GETTEXT` is used entirely to find `libintl` and nothing more, but it seems to really want to drag in lots of other bits, which as far as I can tell this project doesn't need.

Reply:
I've added a comment to configure.ac to note about this